lib/plugins/create/templates/azure-python/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "azure-python",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "serverless-azure-functions": "^1.0.2-19"
```

Comment:
iirc the `^` is ineffective with release tags, hence above equivalent to `1.0.2-19`.
To avoid confusion I would remove `^`

Reply:
Why would my husband use this stuff? I find business stuff. Stock stuff.